README.markdown

```diff
@@ -95,8 +95,13 @@ for better security. To do that you need public and private RSA keys, which can
 following commands:
 
 ```bash
+# RSA
 openssl genrsa -out private-key.pem 1024
 openssl rsa -in private-key.pem -out public-key.pem -pubout
+
+# Ed25519
+openssl genpkey -algorithm ed25519 -out private-key.pem
+openssl pkey -in private-key.pem -out public-key.pem -pubout
```

Comment:
I think also this section of README should be updated:


> Don't forget to put your public key to `foo.bar._domainkey.example.com` TXT DNS record as smth like
> ```
> v=DKIM1; g=*; k=rsa; p=MIGfMA0GCSqGSIb3DQEBA......
> ```
> See RFC6376 for more details.

To mention rfc8463 and that in the DNS should be `k=ed25519` for this new algorithm.

Reply:
Of course.